client/card-readers/settings/sections/business-details.js

```diff
@@ -31,11 +38,21 @@ const BusinessDetailsSection = () => {
 				value={ accountBusinessName }
 				onChange={ setAccountBusinessName }
 			/>
+			{ businessSuppotURLErrorMessage && (
+				<Notice status="error" isDismissible={ false }>
+					<span
+						dangerouslySetInnerHTML={ {
```

Comment:
Is it possible to reduce businessSuppotURLErrorMessage to a plain text format so that the use of `dangerouslySetInnerHTML` may be avoided?

Reply:
The same goes for businessSuppotEmailErrorMessage and businessSuppotPhoneErrorMessage.

Reply:
Sorted.